test/Dialect/FIRRTL/errors.mlir

```diff
@@ -2276,21 +2281,21 @@ firrtl.circuit "InstanceOfClass" {
 // -----
 
 firrtl.circuit "InputProbePublic" {
-  // expected-error @below {{input probe not allowed on public module}}
+  // expected-error @below {{input probe not allowed}}
```

Comment:
This could add a test of input probe on private module being disallowed, too.

Reply:
Done!